packages/excalidraw/element/binding.ts

```diff
@@ -1246,6 +1246,9 @@ export const fixBindingsAfterDuplication = (
     .filter(({ id }) => allBindableElementIds.has(id))
     .forEach((bindableElement) => {
       const oldElementId = duplicateIdToOldId.get(bindableElement.id);
+      if (!oldElementId) {
+        return; //if no bound element is found
+      }
```

Comment:
typescript is doing us really dirty here and I don't know why

Reply:
actually nvm, I just noticed the assertion below

Reply:
can you pls remove that assertion as well?

Reply:
so now do you want me to change anything or not?

Reply:
by assertion I meant on the `find()` call as it's still unsafe like this